gradle/libs.versions.toml

```diff
@@ -59,7 +57,6 @@ junit = { module = "junit:junit", version.ref = "junit" }
 kotlinx-coroutines = { module = "org.jetbrains.kotlinx:kotlinx-coroutines-android", version.ref = "kotlinx-coroutines" }
 okhttp-base = { module = "com.squareup.okhttp3:okhttp", version.ref = "okhttp" }
 okhttp-brotli = { module = "com.squareup.okhttp3:okhttp-brotli", version.ref = "okhttp" }
-okhttp-coroutines = { module = "com.squareup.okhttp3:okhttp-coroutines", version.ref = "okhttp" }
```

Comment:
I left this here on purpose to remember removing our `Call.executeAsync` extension function once OkHttp 5.0 stable is released (last is [5.0.0-alpha.14](https://central.sonatype.com/artifact/com.squareup.okhttp3/mockwebserver/5.0.0-alpha.14)). At that point we have to get rid of:

https://github.com/bitfireAT/icsx5/blob/0ad5697a8cf8e5f176044cd10fab4be618bd1bfe/app/src/main/java/at/bitfire/icsdroid/AsyncUtils.kt#L10-L32

And import the `okhttp-coroutines` library.

However, if you feel that we can just remove this, and remember to do this when 5.0 is released, I'm in for it as well.

Reply:
Oh, ok. We can totally do that. I will add a comment.